### ISSUE TITLE
add_issues_to_epic NotImplementedError: Jira Agile Public API does not support this request

### ISSUE THREAD

**Issue by tophertimzen** [pycontribs/jira#1397]
### Bug summary

`add_issues_to_epic` is no longer working with jira-python. 

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Jira Instance type

Jira Server or Data Center (Self-hosted)

### jira-python version

3.2.0

### Python Interpreter version

3.9

### Which operating systems have you used?

- [X] Linux
- [ ] macOS
- [ ] Windows

### Reproduction steps

```python
jira.add_issues_to_epic(epic, task_keys, True)
```

### Stack trace

```python
File "tool.py, line 83, in morning
    jira.add_issues_to_epic(epic, task_keys, True)
  File "/home/ttimzen/tool/venv/lib/python3.9/site-packages/jira/client.py", line 4868, in add_issues_to_epic
    raise NotImplementedError(
NotImplementedError: Jira Agile Public API does not support this request
```

### Expected behaviour

Should be adding my tasks to an EPIC

### Additional Context

jira==3.2.0

**Comment by studioj** [comment]
@tophertimzen join the discussion here...
Would that solve your problem?
https://github.com/pycontribs/jira/issues/1362

**Comment by studioj** [comment]
@Maffooch would this be able to solve the issue?
https://github.com/pycontribs/jira/issues/1362

**Comment by adehad** [comment]
Fixed in #1351  in 3.3.0
